site/_docs/howto.md

```diff
@@ -990,6 +992,12 @@ address. You can use
 [the 1.20.0 announcement](https://mail-archives.apache.org/mod_mbox/www-announce/201906.mbox/%3CCA%2BEpF8tcJcZ41rVuwJODJmyRy-qAxZUQm9OxKsoDi07c2SKs_A%40mail.gmail.com%3E)
 as a template. Be sure to include a brief description of the project.
 
+Finally, increase the `calcite.version` value in `/gradle.properties` and commit & push
```

Comment:
Give an example commit that people can cherry-pick.

Reply:
Remove 'finally'.

Reply:
Modified.